backup.py

```diff
@@ -50,19 +53,43 @@ def run_rsync(settings, path):
         raise Exception("rsync returned error code {}".format(p.returncode))
 
 
-def run():
+def run_rsync(settings, path):
+    starport = settings.starport
+    cmd = rsync_cmd(settings.ssh_key_path, path, starport)
+    run_cmd_with_logging(cmd)
+
+
+def run_rsync_from_container(settings, source_volume):
+    starport = settings.starport
+    docker_ssh_key_path = "/etc/id_rsa"
+
+    cmd = ["docker", "run", "--rm", "-i", "-t",
+           "-v", "{}:/{}".format(source_volume, source_volume),
+           "-v", "{}:{}".format(expanduser("~/.ssh/known_hosts"), "/root/.ssh/known_hosts"),
+           "-v", "{}:{}".format(abspath(settings.ssh_key_path), docker_ssh_key_path),
+           "instrumentisto/rsync-ssh"]
```

Comment:
How did you choose this image?

Reply:
Pretty randomly, just from the top google hits for rsync ssh docker image! Its a very simple image, we could also create it ourselves.